src/prototype_room_my.js

```diff
@@ -326,6 +326,57 @@ Room.prototype.executeRoom = function() {
     }
   }
 
+  let carryHelpInterval = config.carryHelpers.ticksUntilHelpCheck;
+  let key = _.findKey(Memory.needEnergyRooms, this.name);
+
+  if (Game.time % carryHelpInterval) {
+    this.memory.energyAvailableSum += this.energyAvailable;
+
+  } else if (Memory.needEnergyRooms === undefined) {
+    Memory.needEnergyRooms = [];
```

Comment:
Maybe a collection? Easier to check for room existence and we could store something, maybe